include/rapidjson/internal/ieee754.h

```diff
@@ -22,7 +22,7 @@ namespace internal {
 
 class Double {
 public:
-    Double() {}
+    Double() : u_(0) {}
```

Comment:
I'm not the most familiar with Union, but from some reading it seems like it'd be safer to initialize `d_` rather than `u_` as older versions of the C/C++ standard only allowed the first declared member of a union to be initialized? Maybe it doesn't matter.

Also, be explicit with the type, passing in `0.0` (a double) rather than `0` (an int).

Reply:
@aawoodwa I've revised it.